letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -319,6 +319,32 @@ else
   BOOTSTRAP_VERSION="Unknown 1"
 fi
 
+PrevBootstrapVersion() {
+  if [ -f $VENV_BOOTSTRAP_VERSION ]; then
+    cat $VENV_BOOTSTRAP_VERSION
+  elif grep -Fqx "$BOOTSTRAP_VERSION" << "UNLIKELY_EOF"
+NO_BOOTSTRAP 1
+BootstrapDebCommon 1
```

Comment:
Yes, I grok that and agree. But I still feel like there could be a simpler way.

One might be to say roughly `if [ ! -f $VENV_BOOTSTRAP_VERSION ]; then echo "$BOOTSTRAP_VERSION" | awk '{print $1, 1}' > "$VENV_BOOTSTRAP_VERSION"; fi;`. Then PrevBootstrapVersion could just unconditionally check the file.

There might need to be changes to how NO_BOOTSTRAP is handled; you wouldn't want that getting written to a file. And you wouldn't want it to re-bootstrap unnecessarily if BOOTSTRAP_VERSION was "NO_BOOTSTRAP 1" (because --no-bootstrap was passed) and then control hit `if [ -d "$VENV_PATH" -a "$BOOTSTRAP_VERSION" != "$(PrevBootstrapVersion)" ]; then`.